cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc

```diff
@@ -0,0 +1,242 @@
+:_content-type: ASSEMBLY
+[id=“cloud-experts-using-aws-ack]
+= Tutorial: Using AWS Controllers for Kubernetes on ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-using-aws-ack
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-21
+//---
+//date: '2022-06-02'
+//title: Using AWS Controllers for Kubernetes (ACK) on ROSA
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Paul Czarkowski
+//  - Connor Wooley
+//---
+
+link:https://aws-controllers-k8s.github.io/community/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from {product-title} (ROSA). With ACK, you can take advantage of AWS-managed services for your applications without needing to define resources outside of the cluster or run services that provide supporting capabilities like databases or message queues within the cluster.
+
+Users can install various ACK operators directly from OperatorHub. This makes it relatively easy to get started and start using it with your applications. This controller is a component of the AWS Controller for Kubernetes project. This project is currently in developer preview.
+
+This tutorial shows you how to use the ACK S3 operator as an example, but can be adapted for any other ACK operator in the OperatorHub of your cluster.
+
+== Prerequisites
+
+* A ROSA cluster
+* A user account with `cluster-admin` privileges
+* You have access to the OpenShift CLI (`oc`).
```

Comment:
I think it's safe to remove the punctuation from the last two bullets for consistency